launcher_go/v2/main.go

```diff
@@ -12,11 +12,12 @@ import (
 )
 
 type Cli struct {
-	Version      kong.VersionFlag `help:"Show version."`
-	ConfDir      string           `default:"./containers" hidden:"" help:"Discourse pups config directory." predictor:"dir"`
-	TemplatesDir string           `default:"." hidden:"" help:"Home project directory containing a templates/ directory which in turn contains pups yaml templates." predictor:"dir"`
-	BuildDir     string           `default:"./tmp" hidden:"" help:"Temporary build folder for building images." predictor:"dir"`
-	BuildCmd     DockerBuildCmd   `cmd:"" name:"build" help:"Build a base image. This command does not need a running database. Saves resulting container."`
+	Version      kong.VersionFlag   `help:"Show version."`
+	ConfDir      string             `default:"./containers" hidden:"" help:"Discourse pups config directory." predictor:"dir"`
+	TemplatesDir string             `default:"." hidden:"" help:"Home project directory containing a templates/ directory which in turn contains pups yaml templates." predictor:"dir"`
+	BuildDir     string             `default:"./tmp" hidden:"" help:"Temporary build folder for building images." predictor:"dir"`
+	BuildCmd     DockerBuildCmd     `cmd:"" name:"build" help:"Build a base image. This command does not need a running database. Saves resulting container."`
+	ConfigureCmd DockerConfigureCmd `cmd:"" name:"configure" help:"Configure and save an image with all dependencies and environment baked in. Updates themes and precompiles all assets. Saves resulting container."`
```

Comment:
I think it will be useful to elaborate on what `dependencies` mean here cause I am personally not sure as well.

Reply:
Maybe `dependencies` is the wrong word -- it's the second half of the precompile, which runs theme update and assets compile steps that are dependent on a running database and shared folders.

Maybe a help copy like this would be more descriptive?

> Configure and save an image with all assets fully precompiled and environment configuration baked in. Saves resulting container.